### PR TITLE
PIM-4186: fix @jjanvier comments

### PR DESCRIPTION
**Pull request by sumbobyboys**

### CODE REVIEW

File: src/Pim/Bundle/CatalogBundle/Filter/ProductValueLocaleFilter.php [hunk 1]
```diff
@@ -22,12 +22,12 @@ public function filterObject($productValue, $type, array $options = [])
             throw new \LogicException('This filter only handles objects of type "ProductValueInterface"');
         }
 
-        $locales   = isset($options['locales']) ? $options['locales'] : [];
+        $localeCodes   = isset($options['locales']) ? $options['locales'] : [];
         $attribute = $productValue->getAttribute();
```
Comment (jmleroux) [inline src/Pim/Bundle/CatalogBundle/Filter/ProductValueLocaleFilter.php]: superflous blank spaces + alignment

File: src/Pim/Bundle/EnrichBundle/Controller/LocaleRestController.php [hunk 2]
```diff
@@ -48,7 +48,7 @@ public function __construct(
     public function indexAction()
     {
         $locales           = $this->localeRepository->getActivatedLocales();
-        $fitleredLocales   = $this->collectionFilter->filterCollection($locales, 'pim:internal_api:locale:view');
+        $fitleredLocales   = $this->collectionFilter->filterCollection($locales, 'pim.internal_api.locale.view');
```
Comment (jmleroux) [inline src/Pim/Bundle/EnrichBundle/Controller/LocaleRestController.php]: filtered

**Comment by jmleroux** [comment]
GTM after comments corrections

**Comment by phaseinducer** [comment]
Awesome, GTM

**Pull request merged**